DESCRIPTION

```diff
@@ -52,15 +30,27 @@ Imports:
     matrixStats,
     purrr,
     rlang,
+    stats,
     tidyr,
     tidyselect
+Suggests: 
+    cowplot,
+    ggplot2,
+    hubExamples,
+    hubVis (>= 0.0.0.9100),
+    knitr,
+    rmarkdown,
+    testthat (>= 3.0.0)
```

Comment:
One of the challenges of submitting to CRAN: they do not like it if you try to submit a package that has dependencies on packages that are not on CRAN (even if they are soft dependencies). 

The solution is to add this line that specifies that CRAN can search for the hub packages in our universe repositories (which I will draft guidelines for this week).

```suggestion
    testthat (>= 3.0.0)
Additional_repositories: https://hubverse-org.r-universe.dev/
```

NOTE: you will get two notes for this submission, so it may take a bit of back and forth with the CRAN team, but we can burn that bridge when we get to it. 

```
* This is a new release.
* Suggests or Enhances not in mainstream repositories:
  hubExamples, hubVis
Availability using Additional_repositories specification:
  hubExamples   yes   https://hubverse-org.r-universe.dev/
  hubVis        yes   https://hubverse-org.r-universe.dev/
```

Reply:
You are most likely aware of this, but `hubUtils` and `distfromq` must go to CRAN before this package can.